data/builtInFeatures.js

```diff
@@ -0,0 +1,120 @@
+// https://github.com/zloirock/core-js
+
+/* eslint-disable quotes */
+module.exports = {
+  // es2015
+  // core-js/fn/map
+
+  // "es6.typed/array-buffer": "typed arrays / ",
+  "es6.typed.data-view": "typed arrays / DataView",
```

Comment:
this should cover

typed arrays / DataView (Int8)
typed arrays / DataView (Uint8)
typed arrays / DataView (Int16)
typed arrays / DataView (Uint16)
typed arrays / DataView (Int32)
typed arrays / DataView (Uint32)
typed arrays / DataView (Float32)
typed arrays / DataView (Float64)